### ISSUE TITLE
Could you update this app to NC26 please

### ISSUE THREAD

**Issue by jet438** [nextcloud/ransomware_protection#148]
could you update this app to be compatable with NC26 please

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/ransomware_protection/issues/144

**Comment by affinityv** [comment]
@jet438 This app is total garbage; I enabled it, got a notification of the server having "deleted" files which were, in fact, not deleted, but were not accessible via NC.  I checked backups to see which files to put back and found rsync saw no differences to sync.  Make sure you have backups and don't bother with this useless app that far too easily catches reasonable file names and file extensions without assessing the content whatsoever as far as I can tell.

**Comment by CodeRoutineDev** [comment]
I have been using this app for years without any problems.

**Comment by affinityv** [comment]
> I have been using this app for years without any problems.

Yes, but what is the point of it?  It may stop uploads of bad files, it won't stop the server holding already held bad files.  It seems to only care about file names and file types and many of the file names are legit for other reasons; that is, they are not malware.

Help me understand why you think this app is worth anything please?

As the readme for the app says, this isn't a substitute for backups; backups are essential.  This tool doesn't seem to help anything other than provide a false sense of security by users.

**Comment by nickvergessen** [comment]
> This tool doesn't seem to help anything other than provide a false sense of security by users.

The scenario this app covers is:
* One of your laptops/computers gets infected by ransomware
* The desktop client on the infected computer would try to upload all the modified files
* Afterwards the desktop clients on all so far **NOT** infected computers would download the encrypted files and potentially get infected as well.

So the idea is to prevent the desktop client from further syncing when it was detected to send such files, so it does not spread.

That is also why uploading via the browser is not blocked, because that is not how ransomware works.

**Comment by Githopp192** [comment]
&NC27 - i think

**Comment by ElSi-DVT** [comment]
&NC28 - probably

**Comment by Jafar412** [comment]
NC29

**Comment by bcutter** [comment]
Likely dead piece of software.

But working (at least up to NC 28 - 29 not tested by myself yet) with the necessary changes mentioned.

**Comment by Thomas-Ganter** [comment]
Dead piece of Software.  
When activated on NC 31 the logs are flooded with Error Messages  --  reproducibly turning this on and of either means tsunami tor piece-and-quiet for the logs.

**Comment by Wotisrv** [comment]
Is this prosject dead? As Thomas-Ganter wrote on NC31 the logs are flooded with Error Messages.

**Comment by bcutter** [comment]
Are you sure the log flooding originates from this app? Can you share logs indicating that?

Because when I upgraded from NC 30 to 31 a few minutes ago, the app was disabled but the log got flooded anyway, but not because of this app but https://github.com/nextcloud/server/issues/51976.
